crates/trigger/src/cli.rs

```diff
@@ -119,15 +120,15 @@ where
 
         // TODO: I assume there is a way to do this with a single let mut loader: Box<dyn Loader>
         // variable instead of the entire executor.
-        let executor: Executor = match self.oci {
+        let executor: Executor = match self.from_registry {
```

Comment:
Man I would really like to get rid of this switch and reunify the lockfile model.  I had guessed that was going to be done as part of this work but time is short!  (This is an observation, rather than a passive-aggressive way of asking you to change it now!)